warehouse/query-core/pom.xml

```diff
@@ -55,17 +55,15 @@
             <artifactId>datawave-core</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>gov.nsa.datawave.contrib</groupId>
-            <artifactId>datawave-in-memory-accumulo</artifactId>
-        </dependency>
-        <!--
```

Comment:
Do we need to uncomment datawave-ingest-core?  It's already included as a transitive dependency of datawave-ws-query.

Reply:
I thought I had added a reply to Ivan's comment but it isn't there. As you stated it is a transitive dependency but it was commented out. From my perspective it should not existed as it is. The dependency should either be removed or explicitly added. I have no issue with removing it from the pom file.

Reply:
I have another round of updates and additions that are ready for review. The pom modifications have been added to these changes.

Reply:
Is that going to be in a separate pull request?  I would just delete this dependency for now, since it's already included.

Reply:
Modified the pom file to remove these dependencies for this pull request since there were other necessary modifications.